src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(new Bulldozer(), new Bulldozer(), new Bulldozer());
+    }

```

Comment:
Don't begin class or method implementation with an empty line. It is unnecessary and doesn't follow Java code style conventions.